sociologia-ruralis.csl

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Sociologia Ruralis</title>
+    <title-short>SocRur</title-short>
+    <id>http://www.zotero.org/styles/sociologia-ruralis</id>
+    <link href="http://www.zotero.org/styles/sociologia-ruralis" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-harvard" rel="template"/>
+    <author>
```

Comment:
Add a link to the style's documentation (even if it's outdated) using link with rel=documentation